ninja/management/commands/export_openapi_schema.py

```diff
@@ -55,14 +65,18 @@ def add_arguments(self, parser: CommandParser) -> None:
             help="Output schema to a file (outputs to stdout if omitted).",
         )
         parser.add_argument(
-            "--indent", dest="indent", default=None, type=int, help="JSON indent"
+            "--indent", dest="indent", default=None, type=int, help="Schema indentation level"
         )
 
     def handle(self, *args: Any, **options: Any) -> None:
         api = self._get_api_instance(options["api"])
         schema = api.get_openapi_schema()
         result = json.dumps(schema, cls=NinjaJSONEncoder, indent=options["indent"])
 
+        if options["format"] == "yaml":
+            data = json.loads(result)
+            result = yaml.dump(data, indent=options["indent"])
```

Comment:
@baseplate-admin Good question! I don't remember the exact details, but it has to do with the order of the YAML output. When loaded via `json` first, the order of the underlying data structure is preserved somehow, so that when PyYAML does its thing, the order of the YAML looks better. Obviously, the order technically doesn't matter, but I personally think it's worth it. Do you think I should add a comment about it?

Reply:
If you need sorting [maybe sort the dictionary](https://realpython.com/sort-python-dictionary/)?

Doing JSON → YAML takes a lot of unnecessary processing.

:)